modules/distribution/src/repository/resources/dbscripts/apimgt/h2.sql

```diff
@@ -133,24 +135,24 @@ CREATE TABLE IF NOT EXISTS IDN_OPENID_REMEMBER_ME (
 );
 
 CREATE TABLE IF NOT EXISTS IDN_OPENID_USER_RPS (
-                  USER_NAME VARCHAR(255) NOT NULL,
-                  TENANT_ID INTEGER DEFAULT 0,
-                  RP_URL VARCHAR(255) NOT NULL,
-                  TRUSTED_ALWAYS VARCHAR(128) DEFAULT 'FALSE',
-                  LAST_VISIT DATE NOT NULL,
-                  VISIT_COUNT INTEGER DEFAULT 0,
-                  DEFAULT_PROFILE_NAME VARCHAR(255) DEFAULT 'DEFAULT',
-                  PRIMARY KEY (USER_NAME, TENANT_ID, RP_URL)
+			USER_NAME VARCHAR(255) NOT NULL,
+			TENANT_ID INTEGER DEFAULT 0,
+			RP_URL VARCHAR(255) NOT NULL,
+			TRUSTED_ALWAYS VARCHAR(128) DEFAULT 'FALSE',
+			LAST_VISIT DATE NOT NULL,
+			VISIT_COUNT INTEGER DEFAULT 0,
+			DEFAULT_PROFILE_NAME VARCHAR(255) DEFAULT 'DEFAULT',
+			PRIMARY KEY (USER_NAME, TENANT_ID, RP_URL)
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04